src/main/java/mil/dds/anet/resources/AdminResource.java

```diff
@@ -57,4 +68,97 @@ public Integer saveAdminSettings(@GraphQLRootContext Map<String, Object> context
     return config.getDictionary();
   }
 
+  /**
+   * If anet-dictionary.yml file is changed manually while ANET is up and running ,this method can
+   * be used to reload the dictionary with new values without restarting the server
+   */
+  @GraphQLMutation(name = "reloadDictionary")
+  public String reloadDictionary(@GraphQLRootContext Map<String, Object> context) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    AuthUtils.assertAdministrator(user);
+    config.reloadDictionary();
+    AnetAuditLogger.log("Dictionary updated by {}", user);
+    return AnetConstants.DICTIONARY_RELOAD_MESSAGE;
+  }
+
+  /**
+   * Clears Domain Users Cache
+   */
+  @GraphQLMutation(name = "clearCache")
+  public String clearCache(@GraphQLRootContext Map<String, Object> context) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    AuthUtils.assertAdministrator(user);
+    return AnetObjectEngine.getInstance().getPersonDao().clearCache();
+  }
+
+  /**
+   * Returns the project version which is saved during project build (See project.version definition
+   * in build.gradle file) Right after project information is written into version.properties file
+   * on startup,it is read and set with AnetConfiguration loadVersion method
+   */
+  @GET
+  @Timed
+  @Path("/projectVersion")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Map<String, Object> getProjectVersion() {
+    return new HashMap() {
+      {
+        put("projectVersion", config.getVersion());
```

Comment:
The requirement was to do so through graphql. We want to minimize the number of separate http requests and to keep graphql as a single point of entry.
Incidently, the dictionary is I believe the only thing retrieved through a rest call, at some point (not in this PR), we should also replace that.

Reply:
I've replaced this rest method with GraphQLQuery.
In fact this is the same reason with already existing rest dictionary method.It is used to load dictionary in client dev-mode.
Please check settings.js under client web-dev folder.
This is why I had set version for dev-mode by calling a "rest" service in the same way .

As you said it may be better to apply another solution with another PR. On dev-mode we should find a way to load dictionary and version both in a different way other than calling rest services.
Just for now , in dev-mode I've just assigned a static string value to version parameter.